workflow-ui/compose-tooling/src/main/java/com/squareup/workflow1/ui/compose/tooling/PreviewViewEnvironment.kt

```diff
@@ -38,9 +34,33 @@ import kotlin.reflect.KClass
   }
 }
 
+/**
+ * Creates and [remember]s a [ViewEnvironment] that has a special [ScreenViewFactoryFinder]
+ * and any additional elements as configured by [viewEnvironmentUpdater].
+ *
+ * The [ScreenViewFactoryFinder] will contain [mainFactory] if specified, as well as a
+ * [placeholderViewFactory] that will be used to show any renderings that don't match
+ * [mainFactory]'s type. All placeholders will have [placeholderModifier] applied.
+ */
+@Composable internal fun previewViewEnvironment(
```

Comment:
nit: `PreviewViewEnvironment`

The same goes for the original/overloaded function above this.

Reply:
I think I'll fix that on main.

Reply:
Oh, nevermind, it's non-public. Might as well do it here.

Reply:
Nope!
<img width="752" alt="Screen Shot 2022-03-02 at 12 21 13 PM" src="https://user-images.githubusercontent.com/1884445/156443223-4f8897f0-ca4f-4d42-b2cb-16d0803d55f3.png">

Reply:
Ah!  Nevermind then! My mistake!

Reply:
Should be called `rememberPreview…`

Reply:
#692